.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,28 @@
+name: CI Tests
+
+on: [push, pull_request]
+
+jobs:
+  ci-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        OS_TYPE: ["centos"]
+        OS_VERSION: [7, 8]
+        OSG_VERSION: [3.5, 3.6]
+    env:
+      OS_TYPE: ${{ matrix.OS_TYPE }}
+      OS_VERSION: ${{ matrix.OS_VERSION }}
+      OSG_VERSION: ${{ matrix.OSG_VERSION }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Before Installation
+        run: |
+          sudo apt-get update
+          echo 'DOCKER_OPTS="-H tcp://127.0.0.1:2375 -H unix:///var/run/docker.sock -s devicemapper"' | sudo tee /etc/default/docker > /dev/null
+          sudo service docker restart
+          sleep 5
+          sudo docker pull centos:centos${OS_VERSION}
```

Comment:
I think this may be unnecessary

Reply:
The whole `Before Installation` step is unnecessary? (Just making sure)

Reply:
Yeah, I think so. Try removing it and seeing if the action in your own repo works. If it does, then we can fully drop this step in the PR.

Reply:
I tried removing `Before Installation` step, and the [workflow](https://github.com/yongleyuan/tarball-client/actions/runs/1068897669) ran successfully.